src/app/workflows/workflows.component.ts

```diff
@@ -49,4 +52,8 @@ export class WorkflowsComponent {
     this.searchPage = this.searchPageUrls.includes(this.urlResolverService.getEntryPathFromUrl());
     this.entryName = this.urlResolverService.getEntryPathFromUrl();
   }
+
+  getEntryTerm(type: newEntryType): string {
```

Comment:
WebStorm is giving me compiler errors in workflow.component.html because you're passing a (Swagger) EntryType to this method. I suggest changing the parameter type to (Swagger) EntryType. Then the next line would make more sense to me -- I didn't get why you are casting a `newEntryType` to a `newEntryType`.

Reply:
Makes sense, I think I got mixed up in the names 😫 I'll also update following you other comment